.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+name: ci
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  automated-tests:
```

This job should be named "release" or similar.
```suggestion
  release:
```
or
```suggestion
  build-release-assets:
```